include/swift/AST/DiagnosticsSema.def

```diff
@@ -2733,6 +2733,8 @@ ERROR(differentiable_attr_unsupported_req_kind,none,
       "layout requirement are not supported by '@differentiable' attribute", ())
 ERROR(differentiable_attr_class_unsupported,none,
       "class members cannot be marked with '@differentiable'", ())
+ERROR(differentiable_attr_stored_prop_unsupported,none,
```

Comment:
```suggestion
ERROR(differentiable_attr_stored_property_unsupported,none,
```

I don't quite feel "prop" is a commonly used acronym for "property", unlike "attr" for "attribute".

Reply:
Sure I can change it! I've used this in the past, and on web development, [jQuery](https://www.c-sharpcorner.com/UPLOADFILE/97FC7A/DIFFERENCE-BETWEEN-PROP-AND-ATTR-IN-JQUERY/) use `prop()` and `attr()` which is the one example i can think of.

However I don't have a strong opinion on this, so prop VS property is fine either way 😄

Reply:
Gotcha. I actually meant that it's not so common in the Swift compiler code base. Thanks for changing it!